README.md

```diff
@@ -151,10 +151,12 @@ const [err, user] = await fastify.to(
 ```
 
 #### Custom error handler
-This plugins also adds a custom error handler which hides the error message in case of `500` errors, instead it returns `Something went wrong`.<br>
+This plugins also adds a custom error handler which hides the error message in case of `500` errors that are being generated by throwed errors, instead it returns `Something went wrong`.<br>
 This is especially useful if you are using *async* routes, where every uncaught error will be sent back to the user *(but do not worry, the original error message is logged as error in any case)*.
 If needed, it can be disabled by setting the option `errorHandler` to `false`.
 
+If you return a `500` error via the API provided on `fastify.httpErrors` and `reply` objects, the custom text is always preserved.
```

Comment:
`the custom text is always preserved.` => `the custom error message is preserved and returned in the HTTP response body`?